runtime/serde/serde-cbor/common/src/aws/smithy/kotlin/runtime/serde/cbor/CborSerializer.kt

```diff
@@ -22,21 +27,27 @@ public class CborSerializer :
     StructSerializer {
     private val buffer = SdkBuffer()
 
+    public fun toHttpBody(): HttpBody = buffer.readByteArray().toHttpBody()
+
```

Comment:
Yes that's correct, each message will be encoded separately in the `CborSerializer` instead of all-at-once. I wasn't able to functionally test this because there's no services that have an input event stream **and** support CBOR at the moment. I was able to test output event streams using CloudWatch Logs and they work as expected.